app/assets/stylesheets/members/index.css.scss

```diff
@@ -41,6 +59,30 @@ $parallax-background-height: $parallax-window-height;
   }
 }
 
+// Mobile
+@include breakpoint($mobile) {
+  .parallax-static-content {
+    margin-top: 0;
+    padding-top: 200px;
+    padding-bottom: 300px;
+    height: 100px;
+    background-color: rgba(20, 20, 20, 0.7);
+    color: #9A9A8A;
```

Comment:
Color literals like `#9A9A8A` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#9A9A8A` should be written as `#9a9a8a`